tutorials/clinical/30_ecog.py

```diff
@@ -134,6 +135,31 @@
     envelope=True).resample(sfreq)
 gamma_info = gamma_power_t.info
 
+# %%
+# Plot Gamma Power on cortical sensors
+# --------------------------------------
+#
+# We will now use evoked gamma power to plot on the cortical surface.
+# Therefore we extract the evoked time sample at 15s and normalize
+# it in a range of 0 to 1 in order to map it using a matplotlib colormap.
+
+gamma_power_vec = np.array(gamma_power_t.to_data_frame().iloc[20, 1:])
+rgba = cm.get_cmap("viridis")
+gamma_power_vec_norm = (gamma_power_vec - np.min(gamma_power_vec)) / \
+                       (np.max(gamma_power_vec) - np.min(gamma_power_vec))
+
+gamma_power_color_vec = [rgba(gp)[:3] for gp in gamma_power_vec_norm]
```

Comment:
To me this is a bit more straightforward, but I'll let you decide if you think it's enough of an improvement to bother changing the variable names (you'd need to update the plot_alignment call at least)

```suggestion
# scale values to be between 0 and 1, then map to colors
gamma_power_vec -= gamma_power_vec.min()
gamma_power_vec /= gamma_power_vec.max()
rgba = cm.get_cmap("viridis")
sensor_colors = gamma_power_vec.map(rgba).tolist()
```
I'd also suggest maybe renaming `gamma_power_vec` to `gamma_power_at_15s`